src/borg/testsuite/__init__.py

```diff
@@ -44,7 +48,7 @@
     has_llfuse = False
 
 # The mtime get/set precision varies on different OS and Python versions
-if 'HAVE_FUTIMENS' in getattr(posix, '_have_functions', []):
+if posix != None and 'HAVE_FUTIMENS' in getattr(posix, '_have_functions', []):
```

Comment:
just `if posix` in this case.

in general: never use `== None` or `!= None`, but `is None` or `is not None`.

Reply:
Done